README.md

```diff
@@ -10,6 +10,8 @@ It depends on `rich` and `kaitaistruct` (runtime) module, then it can be launche
 python -m py.tui.main /path/to/tty
 ```
 
+This fork is the same as the original repository except for TUI is now runs on `pyserial` commands for cross-platform support. 
```

Comment:
I have to remove this before merging, since here is the original repo, not a fork anymore

Reply:
fixed [8ff29bb](https://github.com/Jakeler/bms-parser/pull/6/commits/8ff29bbb855949b004ac9709713b56a5d21a9604)